dbt/include/snowflake/macros/relations/dynamic_table/replace.sql

```diff
@@ -1,12 +1,21 @@
-{% macro snowflake__get_replace_dynamic_table_sql(relation, sql) %}
+{% macro snowflake__get_replace_dynamic_table_sql(relation, sql) -%}
+
+    {%- set dynamic_table = relation.from_config(config.model) -%}
 
     create or replace dynamic table {{ relation }}
-        target_lag = '{{ config.get("target_lag") }}'
-        warehouse = {{ config.get("snowflake_warehouse") }}
+        target_lag = '{{ dynamic_table.target_lag }}'
+        warehouse = {{ dynamic_table.snowflake_warehouse }}
+        {% if dynamic_table.refresh_mode %}
+        refresh_mode = {{ dynamic_table.refresh_mode }}
+        {% endif %}
+        {% if dynamic_table.initialize %}
+        initialize = {{ dynamic_table.initialize }}
+        {% endif %}
+        {% if dynamic_table.comment %}
+        comment = '{{ dynamic_table.comment }}'
+        {% endif %}
```

Comment:
On all other materialization / relation types (and across DWHs), we support users adding object-level `comment` by:
- defining a `description`
- configuring `persist_docs: {relation: true}`

For consistency, it feels like that's what we should do here as well. In fact, I would expect it [to already run here](https://github.com/dbt-labs/dbt-snowflake/blob/a4e22b00c362691db200da2baff9ea12e143d41f/dbt/include/snowflake/macros/materializations/dynamic_table.sql#L88), unless there's a gotcha with initial (create) vs. subsequent (alter) runs.

Reply:
I updated this PR to remove comment-related stuff everywhere. I followed the code path you pointed out and it does appear to already handle comments via the description field.